hadoop-common-project/hadoop-common/pom.xml

```diff
@@ -1015,7 +1031,81 @@
         </plugins>
       </build>
     </profile>
-
+    <!-- profile to use already generated protobuf code using 2.5.0 for aarch64-->
+    <profile>
+      <id>aarch64</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+        <os>
+          <family>linux</family>
```

Comment:
`<family>linux</family>` can be removed.

Reply:
sure